<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApplicationUpdateDialog</class>
 <widget class="QDialog" name="ApplicationUpdateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Редактирование приложения</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Сектор:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Договор:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Тип приложения:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Приложение:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Ссылка:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QLineEdit" name="linkEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QPushButton" name="winScanButton">
     <property name="toolTip">
      <string>Открыть WinScan2PDF для сканирования файла</string>
     </property>
     <property name="text">
      <string>WinScan2PDF</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QPushButton" name="saveButton">
     <property name="toolTip">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Сохранить изменения&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="3" colspan="3">
    <widget class="QPushButton" name="rejectButton">
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="selectFileButton">
     <property name="toolTip">
      <string>Выбрать файл</string>
     </property>
     <property name="text">
      <string>Выбрать файл</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="5">
    <widget class="QLineEdit" name="applicationEdit"/>
   </item>
   <item row="0" column="1" colspan="5">
    <widget class="MyComboBox" name="secCombo" native="true">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="5">
    <widget class="MyComboBox" name="dealCombo" native="true"/>
   </item>
   <item row="2" column="1" colspan="5">
    <widget class="MyComboBox" name="typeCombo" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyComboBox</class>
   <extends>QWidget</extends>
   <header location="global">mycombobox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>applicationEdit</tabstop>
  <tabstop>linkEdit</tabstop>
  <tabstop>selectFileButton</tabstop>
  <tabstop>winScanButton</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>rejectButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
